sql/core/src/test/resources/sql-tests/results/timestampNTZ/timestamp.sql.out

```diff
@@ -373,17 +374,19 @@ struct<to_timestamp(2019-10-06 10:11:12.1234, yyyy-MM-dd HH:mm:ss.SSSSSS[zzz]):t
 -- !query
 select to_timestamp('2019-10-06 10:11:12.12345CST', 'yyyy-MM-dd HH:mm:ss.SSSSSS[zzz]')
```

Comment:
timestamp ntz is not released yet so we still have a chance to change the behavior. I think we do need to fail this? @gengliangwang

Reply:
I think there is a bigger question now and somewhat a throwback at the original design of that function. Should the function for timestamp_ntz fail processing timestamps with timezone or not?

Reply:
![image](https://user-images.githubusercontent.com/1097932/143054054-2cc408d6-0894-4584-9034-4561fe82ab6c.png)

I look it up in the ANSI SQL spec. The string input here can be considered as Timestamp with TZ. I think we should allow casting timestamp string with TZ to TimestampNTZ.

Reply:
OK, then shall we apply the same thing in JSON/CSV? basically, string with timezone can be read as both LTZ and NTZ, so does string without timezone.

Reply:
cc @sadikovi

Reply:
I already reverted this change in the latest version. If we allow strings with timezone to be read as NTZ, then how do we infer the timestamp type correctly? Do you want all timestamps to be converted to `spark.sql.timestampType`?

Reply:
We can make it so depending on the default timestamp type, you would read them as timestamp LTZ or NTZ. It would be different from the design doc though.